internal/jac/jac.go

```diff
@@ -14,10 +14,21 @@ import (
 	"strings"
 )
 
+var dependency = &dependencies.Dependency{
+	Command:    "jac",
+	Url:        "https://github.com/nestoca/jac",
+	IsRequired: false,
+	RequiredBy: []string{"project owners", "release owners"},
+}
+
+func init() {
+	dependencies.Add(dependency)
+}
+
 func ListProjectPeople(catalogDir string, extraArgs []string) error {
-	err := ensureJacCliInstalled()
+	err := dependency.Check()
 	if err != nil {
-		return err
+		os.Exit(1)
 	}
```

Comment:
Would it make sense to add a `MustBePresent` method and exit out there instead of peppering  `if err...`?